test/libsolidity/semanticTests/abiEncoderV2/abi_encode_v2_in_function_inherited_in_v1_contract.sol

```diff
@@ -31,5 +33,5 @@ contract C is B {
 // ----
```

Comment:
@cameel this test is not as useful as it could be, as it doesn't really test a V1 feature, because it has `compileViaYul: also`, i.e. the `pragma abicoder v1` is ignored in that case, yet it passes.

Reply:
If the pragma is ignored, why did the gas update change?

Reply:
When writing it I was not aware that v1 is never used in Yul mode. I think that `compileViaYul: also` does not hurt but it would be fine to just change it to `compileViaYul: false`.

Reply:
> If the pragma is ignored, why did the gas update change?

It's only ignored by the new codegen.